docs/replica-sets.md

```diff
@@ -0,0 +1,117 @@
+# VirtualMachineReplicaSet
+
+## Overview
+
+In order to allow scaling up or down similar VMs, add a way to specify
+`VirtualMachine` templates and the amount of replicas required, to let the
+runtime create these `VirtualMachine`s.
+
+## Use-cases
+
+### Ephemeral VMs
+
+Scaling ephemeral VMs which only have read-only mounts, or work with a backing
+store, to keep temporary data, which can be deleted after the VM gets
+undefined.
+
+## Implementation
+
+A new object `VirtualMachineReplicaSet`, backed with a controller will be
+implemented:
+
+```yaml
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachineReplicaSet
+metadata:
+  name: myreplicaset
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      mylabel: mylabel
+  template:
+    metadata:
+      name: test
+      labels:
+        mylabel: mylabel
+    spec:
+      domain:
+        devices:
+      [...]  
+```
+
+`spec.template` is similar to a `VirtualMachineSpec`. `spec.replicas` specifies
```

Comment:
How `similar` is it? Wort specifying what it can/can not contain?

Reply:
Actually not just similar, it is the same. Will fix the docu.

Reply:
done